### PR TITLE
grants: add harmony support

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- [x] Grant Creation
- [x] Grant Detail Page (View + Update)
- [x] Add to Cart
- [x] Contribute Flow 
   - when one wallet extension is not installed
   - when not enough balance
   - when wallet is locked
   - when sign is rejected
   - when signature is approved
- [x] Chain validation + sending email on successful validation 

https://share.vidyard.com/watch/E47CWxua89CM8xM5PtcCLA?

### CODE REVIEW

File: app/grants/views.py [hunk 1]
```diff
@@ -3048,12 +3048,12 @@ def contribute_to_grants_v1(request):
             })
             continue
 
-        if is_grant_team_member(grant, profile):
-            invalid_contributions.append({
-                'grant_id': grant_id,
-                'message': 'error: team members cannot contribute to own grant'
-            })
-            continue
+        # if is_grant_team_member(grant, profile):
```
Comment (chibie) [inline app/grants/views.py:3051]: why is this commented?

**Review by chibie — CHANGES_REQUESTED** [review]

File: app/grants/views.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/views.py:3051]: cause i don't think should be present at this point ! commenting it out and will take it out later if folks are good with it

File: app/assets/v2/js/cart.js [hunk 2]
```diff
@@ -371,6 +374,9 @@ Vue.component('grants-cart', {
         case 'ZIL':
           vm.chainId = '102';
           break;
+        case 'HAROMONY':
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/assets/v2/js/cart.js]: oh thanks for that ^_^

File: app/grants/views.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/views.py:3051]: this is most likely being removed in my follow up PR with the polkadot stuff

**Review by chibie — APPROVED** [review]

File: app/grants/views.py (hunk 1 above)
Comment (octavioamu) [reply app/grants/views.py:3051]: but we are not longer validate if the contributor is also part of the grant?

**Review by octavioamu — APPROVED** [review]
looking good the only thing I noticed is the harmony tab is trying to connect with web3modal

**Comment by thelostone-mc** [comment]
> looking good the only thing I noticed is the harmony tab is trying to connect with web3modal

Yup this is something we'll have to fix for all non web3 related flows

**Pull request merged**